dev-deployer-nixops/installBlockPerf.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+# Make folder for blockperf
+cd /run/keys
+
+# Make sure we do a fresh install
+rm -rf blockperf
+
+# Pull repo
+git clone https://github.com/cardano-foundation/blockperf.git
+
+# Checkout networking team branch
+cd blockperf
+git checkout bolt12/network-team-patch
```

Comment:
We need that patch until we have a working nixified blockperf integrated in the cardano-node nixos service

Reply:
Using a blockperf nix flake package with a nix fetchPatch would be an alternative here, but since the flake packaging isn't finalized yet, as well as possibly other reasons, I understand that the venv approach is preferred at the moment.